### PR TITLE
Update triage board design

### PR DESCRIPTION
**Pull request by erikh360**
based on 
https://drive.google.com/file/d/1OQi7jsQAx7zkOOR8rPITv7Qr47TYykvY/view

less columns because of the meeting here:
https://docs.google.com/document/d/1JPUfPlzeJtx-f6hVEdCsueDganoeNwEjb-eCJZOEAog/edit

Local example:
Screenshot 2019-05-31 at 14 47 40

### CODE REVIEW

File: cspatients/models.py [hunk 1]
```diff
@@ -16,6 +16,20 @@ def __str__(self):
...
+    COLD = 2
+    HOT_YELLOW = 3
+    HOT_ORANGE = 4
+    IMMEDIATE = 5
+
+    URGENCY_CHOICES = (
+        (ELECTIVE, "Elective"),
+        (COLD, "Cold"),
+        (HOT_YELLOW, "Hot"),
+        (HOT_ORANGE, "Hot"),
```
Comment (nathanbegbie) [inline cspatients/models.py:29]: Should these have the same key?

**Review by nathanbegbie — CHANGES_REQUESTED** [review]
Just not sure about the repetition of the db choices.

Also, are we still needing to make this view mobile-friendly? If so, would you mind posting a screenshot of what it looks like on mobile?

File: cspatients/models.py (hunk 1 above)
Comment (erikh360) [reply cspatients/models.py:29]: Yeah hot yellow and hot orange. hehe
hot_hot

**Comment by erikh360** [comment]
@nathanbegbie This is on a iPhone plus older gen screen via google dev tools:
mobileview

Smaller screens will have a little bit of side scroll. This is why we removed some fields and we decided a little scrolling is not a big deal.

**Review by nathanbegbie — APPROVED** [review]

**Pull request merged**